site/guide/manage-roles-and-permissions.qmd

```diff
@@ -0,0 +1,55 @@
+---
+title: "Manage roles & permissions"
+keywords: "manage roles, manage permissions, ValidMind"
+---
+
+These steps guide you through the process of configuring and overseeing roles and permissions in the ValidMind Platform UI. By carefully defining what users can and cannot do, you can ensure that the right individuals have the appropriate level of access to resources and can perform the correct actions. 
+
+## Prerequisites
+
+To perform these steps, you must be a group administrator or a superuser. 
+
+## Add new roles
+
+To add a new role:
+
+1. [Log in to the ValidMind UI]({{< var vm_ui >}}).
+
+2. Under **User Permissions** in the sidebar, select **Roles & Permissions**.
+
+3. Click on the **Add New Role** button to initiate the role creation process.
+   - Specify a name for the new role.
+   - Define the role's description and purpose.
+   - Assign permissions to the role based on the actions it should be allowed to perform.
+   - Save the new role configuration.
+
+4. Verify that the new role appears in the list of available roles, ensuring it is correctly set up for use.
+
+## Manage permissions
+
+To manage the permissions for existing roles:
+
+1. [Log in to the ValidMind UI]({{< var vm_ui >}}).
+
+2. Navigate to the **Roles & Permissions** page.
```

Comment:
I don't think this page actually exists? Maybe **User Directory** but please verify in the UI.

Reply:
Can we hold on updating the Manage Roles as well, need to confirm whether it's within scope? Let me work more closely with the developers on completing this bit and I can update the content better. Potentially yes removing the page seems likely at this moment.

Reply:
Concluded that main changes will not include this page.